request_logger.go

```diff
@@ -0,0 +1,76 @@
+package xecho
+
+import (
+	"math"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/sirupsen/logrus"
+)
+
+type TimeProvider func() time.Time
+
+func RequestLoggerMiddleware(timeFn TimeProvider) echo.MiddlewareFunc {
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		return EchoHandler(func(c *Context) error { return RequestLogger(c, next, timeFn) })
+	}
+}
+
+func RequestLogger(c *Context, next echo.HandlerFunc, time TimeProvider) error {
```

Comment:
how come this fn is now exported? I've just keep it in the `RequestLoggerMiddleware` as before